tests/valWithDynOuter/valWithDynOuter.fz

```diff
@@ -99,16 +99,6 @@ valWithDynOuter is
     redef a => "t"
     redef x => "$f-$a:t.x"
 
-/* NYI: this does not compile yet
```

Comment:
`redef e` is redefining a constructor. So it is correct that this does not compile